addons/api/mirage/factories/target.js

```diff
@@ -25,6 +25,7 @@ export default factory.extend({
    * -1 means "unlimited" and we want to generate these on occasion.
    */
   session_connection_limit: () => random.arrayElement([-1, datatype.number()]),
+  worker_filter: (i) => (i % 2 === 0 ? random.words() : null),
```

Comment:
Do we generally try and return different scenarios for the data?

Reply:
As of right now, I do not think it is used anywhere else in the UI. It is an optional field but we weren't sure if it would be worth while to verify that it shows up properly when it is present vs when it is left null.